dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -37,6 +37,41 @@ const moveToTestPanel = () => {
   cy.get('h1').contains(TEST_PANEL).should('exist');
   cy.wait(delay);
 };
+var i=1;
```

Comment:
use let instead of var. 

Spaces around `=` sign are not consistent throughout the file. Somewhere you have put spaces before and after `=` and at some places, you have not given any spaces. According to the generic style guidelines, there should be a space before and after the `=`.

Reply:
Changed var to let and given spaces before and after =